earthly/postgresql/scripts/std_docs.py

```diff
@@ -0,0 +1,438 @@
+#!/usr/bin/env python3
+
+# cspell: words dbmigrations dbviz dbhost dbuser dbuserpw Tsvg
+
+from typing import Optional
+import python.cli as cli
+import python.db_ops as db_ops
+import argparse
+import rich
+from rich import print
+import os
+import re
+import json
+from dataclasses import dataclass
+from textwrap import indent
+
+
+@dataclass
+class DiagramCfg:
+    title: str
+    version: int
+    migration_name: str
+    tables: Optional[list[str]]
+    included_tables: Optional[list[str]]
+    excluded_tables: Optional[list[str]]
+    comments: Optional[bool]
```

Comment:
Since comments is whether boolean or None, might be better to name it to something like is_comments?

Reply:
Would cause unnecessary refactor (even if small) in a utility script.